Dockerfile

```diff
@@ -62,6 +62,13 @@ RUN ./install-packages.sh \
   vim \
   nano \
   tree \
+  clang \
+  \
+  bear \
+  clang-tidy \
+  pylint3 \
+  flake8 \
+  mypy \
```

Comment:
I would expect the Python tooling to be brought in by boots and our Python requirements.

Reply:
where in boots should these packages be added?